.github/actions/compute-matrix/compute-matrix.sh

```diff
@@ -27,6 +32,11 @@ extract_matrix() {
   write_output "PER_CUDA_COMPILER_MATRIX"  "$per_cuda_compiler_matrix"
   write_output "PER_CUDA_COMPILER_KEYS" "$(echo "$per_cuda_compiler_matrix" | jq -r 'keys | @json')"
 
+  local nvcc_full_matrix_minstd_17="$(echo "$matrix" | jq -cr '.nvcc' | filter_minstd_17 | explode_std_versions)"
```

Comment:
suggestion: Alternatively, we could just define a separate sub-matrix for CUDA next testing. That would let us limit which host compilers we test it with as well.

Reply:
Let's discuss at the meeting. I'm going off of the assumption that we want to treat it like our other libraries with the same level of testing, but if we need to reduce coverage that's definitely an option.